### PR TITLE
Home page changes

### PR DESCRIPTION
**Pull request by niraj-scaletech**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,37 +68,37 @@ <h1>DataMiner at a glance</h1>
...
 				<h3>Portable</h3>
-				<p>DataMiner runs on Linux, macOS, Windows, and all popular hardware architectures. It has idiomatic client APIs for major programming languages.</p>
+				<p>Cross-platform compatibility with Linux, macOS, Windows, and all major hardware architectures, featuring idiomatic client APIs for major programming languages.</p>
 				<a href="why_dataminer#portable" class="textbutton arrow-right">Read more</a>
 			</div>
 			<div class="featurebox">
 				<div class="icon featurerich"></div>
 				<h3>Feature-rich</h3>
-				<p>DataMiner offers a rich SQL dialect. It can read and write file formats such as CSV, Parquet, and JSON, to and from the local file system and remote endpoints such as S3 buckets.</p>
+				<p>DataMiner features a robust SQL dialect and supports reading and writing CSV, Parquet, and JSON formats, both locally and from remote endpoints like S3 buckets.</p>
```
Comment (dharmendrasha) [inline index.html:83]: we are not a sql company we are a data company we line should be related to data not sqs try chat GPT

File: index.html [hunk 2]
```diff
@@ -68,37 +68,37 @@ <h1>DataMiner at a glance</h1>
...
 				<h3>Fast</h3>
-				<p>DataMiner runs analytical queries at blazing speed thanks to its columnar engine, which supports parallel execution and can process larger-than-memory workloads.</p>
+				<p>DataMiner executes analytical queries with exceptional speed, leveraging its columnar engine, which supports parallel execution and handling large-scale workloads that exceed memory capacities efficiently.</p>
 				<a href="why_dataminer#fast" class="textbutton arrow-right">Read more</a>
 			</div>
 			<div class="featurebox">
 				<div class="icon extensible"></div>
 				<h3>Extensible</h3>
-				<p>DataMiner is extensible by third-party features such as new data types, functions, file formats and new SQL syntax.</p>
+				<p>DataMiner's flexibility extends through integration of third-party features like novel data types, functions, file formats, and innovative SQL syntax.</p>
```
Comment (dharmendrasha) [inline index.html:95]: same here we are not sql company

File: index.html [hunk 3]
```diff
@@ -68,37 +68,37 @@ <h1>DataMiner at a glance</h1>
...
 				<a href="why_dataminer#extensible" class="textbutton arrow-right">Read more</a>
 			</div>
 			<div class="featurebox">
 				<div class="icon free"></div>
 				<h3>Free</h3>
-				<p>DataMiner and its core extensions are <nobr>open-source</nobr> under the permissive MIT License.</p>
+				<p>DataMiner, along with its core extensions, is freely available as <nobr>open-source</nobr> under the permissive MIT License.</p>
```
Comment (dharmendrasha) [inline index.html:101]: we are not under MIT license it proppriety

**Review by dharmendrasha — APPROVED** [review]
nice please update few lines

File: index.html (hunk 3 above)
Comment (niraj-scaletech) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (niraj-scaletech) [reply index.html]: done

**Pull request merged**